device/globals/game.gd

```diff
@@ -357,29 +357,26 @@ func set_inventory_enabled(p_enabled):
 
 func set_camera_limits():
 	var cam_limit = camera_limits
-	if camera_limits.size.x == 0 && camera_limits.size.y == 0:
-		var p = get_parent()
+	if camera_limits.size.x == 0 and camera_limits.size.y == 0:
 		var area = Rect2()
-		for i in range(0, p.get_child_count()):
-			var c = p.get_child(i)
-			if c is preload("res://globals/background.gd"):
-				var pos = c.get_global_position()
-				var size = c.get_size()
+		for child in get_parent().get_children():
+			if child is preload("res://globals/background.gd"):
+				var pos = child.get_global_position()
+				var size = child.get_size()
 				area = area.expand(pos)
 				area = area.expand(pos + size)
-			if c is preload("res://globals/background_area.gd"):
-				var pos = c.get_global_position()
-				var size = c.get_texture().get_size()
+			if child is preload("res://globals/background_area.gd"):
+				var pos = child.get_global_position()
+				var size = child.get_texture().get_size()
 				area = area.expand(pos)
 				area = area.expand(pos + size)
 
 		camera.limit_left = area.position.x
 		camera.limit_right = area.position.x + area.size.x
-		var cam_top = area.position.y # - main.screen_ofs.y
-		camera.limit_top = cam_top
-		camera.limit_top = cam_top + area.size.y + main.screen_ofs.y * 2
```

Comment:
I'm not sure what the purpose of `main.screen_ofs` is, but this puts the camera outside of the background.

Reply:
ofs usually means "offset" so that makes sense :)

Reply:
Hmm, why would that have been desirable?

Reply:
Maybe a dirty hack for Dog Mendoza? @punto-  ?

Reply:
Probably a dirty hack for Dog Mendoza. Let's ask @punto- , but I think this can go away.